test/WebJobs.Script.Tests/Managment/VirtualFileSystemFacts.cs

```diff
@@ -188,18 +236,18 @@ private static HttpRequest CreateRequest(string path)
         {
             var r = new DefaultHttpContext().Request;
             r.Host = new HostString("localhost");
-            r.Path = new PathString("/admin/vfs" + path);
+            r.Path = new PathString("/admin/vfs/site/wwwroot" + path);
```

Comment:
Fixing this up since by default all requests must be absolute paths (try yourself in production :))